apps/liquidity-provision-dashboard/src/app/components/dashboard/market-list/market-list.tsx

```diff
@@ -0,0 +1,176 @@
+import { useCallback, useState } from 'react';
+import { AgGridColumn } from 'ag-grid-react';
+import type {
+  GroupCellRendererParams,
+  ValueFormatterParams,
+  GetRowIdParams,
+  RowClickedEvent,
+} from 'ag-grid-community';
+import 'ag-grid-community/dist/styles/ag-grid.css';
+import 'ag-grid-community/dist/styles/ag-theme-alpine.css';
+import { formatNumber, t } from '@vegaprotocol/react-helpers';
+import { Icon, AsyncRenderer } from '@vegaprotocol/ui-toolkit';
+import type { Market } from '@vegaprotocol/liquidity';
+import {
+  useMarketsLiquidity,
+  formatWithAsset,
+  displayChange,
+} from '@vegaprotocol/liquidity';
+import type { MarketTradingMode } from '@vegaprotocol/types';
+
+import { HealthBar } from '../../health-bar';
+import { Grid } from '../../grid';
+import { HealthDialog } from '../../health-dialog';
+import { Status } from '../../status';
+
+const marketNameCellRenderer = ({
+  value,
+  data,
+}: {
+  value: string;
+  data: Market;
+}) => {
+  return (
+    <>
+      <span style={{ lineHeight: '12px' }}>{value}</span>
+      <span style={{ lineHeight: '12px' }}>
+        {data?.tradableInstrument?.instrument?.product?.settlementAsset?.symbol}
+      </span>
+    </>
+  );
+};
+
+const healthCellRenderer = ({
```

Comment:
chore (blocking): This is just a component so lets upper case it to follow convention. If you then can't pass it to ag grid Id wrap it with a function.

```
cellRenderer={(props) => <HealthCell {...props} />}
```